grin/src/AbstractInterpretation/EffectTracking/Result.hs

```diff
@@ -0,0 +1,81 @@
+{-# LANGUAGE LambdaCase, RecordWildCards, TemplateHaskell, GeneralizedNewtypeDeriving, TypeFamilies, DeriveFunctor, ViewPatterns #-}
+module AbstractInterpretation.EffectTracking.Result where
+
+import Data.Int
+import Data.Set (Set)
+import qualified Data.Set as Set
+import Data.Map (Map)
+import qualified Data.Map as Map
+import Data.Vector (Vector)
+import qualified Data.Vector as V
+import qualified Data.Bimap as Bimap
+
+import Data.Maybe
+
+import Lens.Micro.Platform
+import Lens.Micro.Extra
+
+import Grin.Grin (Name, Tag, External(..))
+import Grin.Pretty
+import AbstractInterpretation.IR hiding (Tag, SimpleType)
+import AbstractInterpretation.Reduce (ComputerState(..))
+import qualified Grin.TypeEnv as TypeEnv
+import qualified AbstractInterpretation.IR as IR
+import qualified AbstractInterpretation.Reduce as R
+import AbstractInterpretation.EffectTracking.CodeGenBase (ETMapping(..))
+
```

Comment:
Nit: I tend to leave two or three newlines between the last import and the first definition in a function. It is more pleasant to the eye.

Reply:
Done.